src/config.c

```diff
@@ -2140,6 +2140,9 @@ static int updateMaxmemory(const char **err) {
         }
         performEvictions();
     }
+    /* The function is called via 'CONFIG SET maxmemory', we don't want to propagate it
+     * because server.dirty might have been incremented by performEvictions() */
+    preventCommandPropagation(server.current_client);
```

Comment:
please help me follow this.
did we ever propagated CONFIG SET to replicas? or is this a new concern added by this PR?

Reply:
yes, we did
form top comment:
```
CONFIG SET can create evictions, sending notifications which can cause to dirty++ with modules.
we need to prevent it from propagating to AOF/replicas
```

Reply:
ok, so that bug (propagating CONFIG SET) is not new (probably exists in previous releases), but only happens when modules listen to the "evicted" notification, and do some RM_Call or other propagation from within that callback. right?
So i guess this doesn't mean we wanna backport such a fix, but we may still wanna mention it in the top comment.

Reply:
right